tests/DTS/UseCase/Render/AddHotRenderRequestTest.php

```diff
@@ -4,44 +4,35 @@
 namespace HomeCEU\Tests\DTS\UseCase\Render;
 
 
+use HomeCEU\DTS\UseCase\Exception\InvalidRequestException;
 use HomeCEU\DTS\UseCase\Render\AddHotRenderRequest;
-use HomeCEU\Tests\DTS\TestCase;
-use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
 
 class AddHotRenderRequestTest extends TestCase {
```

Comment:
I didn't realize this test existed so I wrote one and then I found this (not sure how I missed it). I don't need to test BuildFromState here as that's tested by the AbstractEntity test, and the other tests in this class.